pkg/platform/model/buildplanner.go

```diff
@@ -53,7 +52,7 @@ type BuildResult struct {
 	CommitID            strfmt.UUID
 	Build               *bpModel.Build
 	BuildStatusResponse *headchef_models.V1BuildStatusResponse
-	BuildStatus         headchef.BuildStatusEnum
+	BuildStatus         string
 	BuildReady          bool
```

Comment:
I wonder if `BuildReady` is necessary now that we can just test that `BuildStatus == bpModel.Complete`.